docs/UsersGuide/source/VXCases.rst

```diff
@@ -0,0 +1,303 @@
+----------------------
+
+Navigate (``cd``) to the experiment directory (``$EXPTDIR``) and run the launch script:
+
+.. code-block:: console
+
+   ./launch_FV3LAM_wflow.sh
+
+Run the launch script regularly and repeatedly until the experiment completes. 
+
+To check progress, run:
+
+.. code-block:: console
+
+   tail -n 40 log.launch_FV3LAM_wflow
+
+Users who prefer to automate the workflow via :term:`crontab` or who need guidance for running without the Rocoto workflow manager should refer to :numref:`Section %s <Run>` for these options. 
+
+If a problem occurs and a task goes DEAD, view the task log files in ``$EXPTDIR/log`` to determine the problem. Then refer to :numref:`Section %s <RestartTask>` to restart a DEAD task once the problem has been resolved. For troubleshooting assistance, users are encouraged to post questions on the new SRW App `GitHub Discussions <https://github.com/ufs-community/ufs-srweather-app/discussions/categories/q-a>`__ Q&A page. 
+
+Generate Plots
+---------------
+
+The plots are created using the graphics generation script that comes with the SRW App v2.1.0 release. Information on the plots and instructions on how to run the script can be found in :doc:`Chapter 12 <srw_v2.1.0:Graphics>` of the v2.1.0 release documentation. If the python environment is already loaded (i.e., ``(regional_workflow)`` is visible in the command prompt), users can navigate to the directory with the plotting scripts and run ``plot_allvars.py``:
+
+.. code-block:: console
+
+   cd <path/to/ufs-srweather-app/ush/Python>
+   python plot_allvars.py 2019061500 0 60 6 </path-to/$EXPTDIR> /<path-to/NaturalEarth> SUBCONUS_Ind_3km
+
+Compare
+===========
+
+Once the experiment has completed (i.e., all tasks have "SUCCEEDED" and the end of the ``log.launch_FV3LAM_wflow`` file lists "Workflow status: SUCCESS"), users can compare their forecast results against the forecast results provided in the ``Indy-Severe-Weather`` directory downloaded in :numref:`Section %s <GetSampleData>`. This directory contains the forecast output and plots from NOAA developers under the ``postprd`` directory and METplus verification files under the ``metprd`` directory. 
+
+Qualitative Comparision of the Plots
+----------------------------------------
+
+Comparing the plots is relatively straightforward since they are in ``.png`` format, and most computers can render them in their default image viewer. :numref:`Table %s <AvailablePlots>` lists plots that are available every 6 hours of the forecast (where ``hhh`` is replaced by the three-digit forecast hour): 
+
+.. _AvailablePlots:
+
+.. table:: Sample Indianapolis Forecast Plots
+
+   +-----------------------------------------+-----------------------------------+
+   | Field                                   | File Name                         |
+   +=========================================+===================================+
+   | Sea level pressure                      | slp_conus_fhhh.png                |
```

Comment:
The png filenames changed from 'conus' to 'regional'. Example: slp_regional_fhhh.png
We did this so the background of the map will match with the forecast regional domain.